functional-samples/tutorial.tabs-manager/README.md

```diff
@@ -0,0 +1,12 @@
+# Tab Manager For Chrome Dev Docs
+
+This Chrome extension helps you manage and group your tabs related to Google Chrome Developer documentation. It organizes tabs into a group titled "DOCS" for easier navigation and enhanced productivity.
+
+Purpose of this extension is to demonstrate [chrome.tabsGroup](https://developer.chrome.com/docs/extensions/reference/api/tabGroups) API to interact with the browser's tab grouping system.
```

Comment:
```suggestion
Purpose of this extension is to demonstrate [chrome.tabsGroups](https://developer.chrome.com/docs/extensions/reference/api/tabGroups) API to interact with the browser's tab grouping system.
```

Reply:
Fixed typo, it should be tabGroups instead of tabsGroup.